app/types/redux.ts

```diff
@@ -21,7 +21,7 @@ export interface NodeState {
 }
 
 export interface WalletState {
-  walletFiles: Array<string>;
+  walletFiles: Array<string> | null;
```

Comment:
Why do we need `Nullable` here? What is the difference with an empty array?

Reply:
empty array can represent 2 different states: initial state and no wallet files found after scanning app folder
app has to show loader while scanning so as long as walletFiles is set to null - scanning in progress